x-team/auto-ui
# Add bypass class names for applicant badge actions

**Release Type:** *New feature* 


Fixes x-team/auto#1432

## Description

- Adds the bypass class names for applicant badge actions.

## Checklist



## Steps to Test or Reproduce

- Navigate to the `ApplicantBadge` component section.
  - Make sure the component passes regression.
- Navigate to the `Dropdown` component section.
  - Make sure the component passes regression.
- Navigate to the `SelectBox` component section.
  - Make sure the component passes regression.

## Related PRs

branch | PR
------ | ------
`1432-fix-style-for-applicants-exclusion-list-dropdown` | [Auto](https://github.com/x-team/auto/pull/1467)
